test/mappings.test.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as util from './testUtil';
+
+suite('YAML mappings', () => {
+  test('Flow mappings as keys in mapping', function () {
+    const content = `
+    foo:
+      { bar: baz }:
+        - value
+      { qux: quux }:
+        - value
+    `;
+
+    util.testErrors(content, []);
```

Comment:
I'm sure I'm missing something, but I do not see what code this tests. What is the point of these tests?

Reply:
Previously the content would error with duplicate key because scalars were the only thing allowed. This now checks that flow mapping won't throw errors and that objects can be the values as well. I didn't know this but `util.testErrors(content, []);` automatically asserts inside of it and does the checking. So in this case its basically asserting that no errors are found when it parses

Reply:
Interesting, thanks.